src/v/cloud_topics/level_zero/stm/placeholder.h

```diff
@@ -17,27 +17,46 @@
 #include "serde/rw/named_type.h"
 #include "serde/rw/uuid.h"
 
+/*
+ * A placeholder is a batch that is replicated into the CTP with its payload
+ * replaced by an address to where the payload is stored. For example, if a
+ * client application produces a batch with a 1 MB payload, a small placeholder
+ * batch may be replicated to the CTP (~100 bytes) with the larger payload
+ * stored in object storage.
+ *
+ * Structure of placeholder batch
+ * ==============================
+ *
+ * - Header: logically identical to the header which would exist if the payload
+ * had been replicatedl. Physically, fields such as checksums and size will
```

Comment:
typo: replicatedl

Reply:
lol no it's fine to leave

Reply:
cool. i'll split the bonus check for saving the money with you, which i'm assuming we'll receive any day now.